go.mod

```diff
@@ -223,7 +223,7 @@ require (
 	go.uber.org/multierr v1.6.0 // indirect
 	go4.org v0.0.0-20201209231011-d4a079459e60 // indirect
 	golang.org/x/mod v0.4.2 // indirect
-	golang.org/x/net v0.0.0-20210525063256-abc453219eb5 // indirect
+	golang.org/x/net v0.0.0-20211209124913-491a49abca63 // indirect
```

Comment:
Does this survive things like `go mod tidy`? The `replace` isn't necessary?

Reply:
It does, but it causes some other change, which I just pushed.